cards-template1a.js

```diff
@@ -2,19 +2,39 @@
 
```

Comment:
Remember when cloning the repository in git to create your own folder (named after yourself!) to store your own homework solutions in.